plugins/Toolbox/resources/qml/ToolboxAuthorPage.qml

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Toolbox is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Dialogs 1.1
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import UM 1.1 as UM
+
+Item
+{
+    id: base
+    property var details: toolbox.authorsModel.items[0]
+    anchors.fill: parent
+    ToolboxBackColumn
+    {
+        id: sidebar
+    }
+    Rectangle
+    {
+        id: header
+        anchors
+        {
+            left: sidebar.right
+            right: parent.right
+            rightMargin: UM.Theme.getSize("double_margin").width
+        }
+        height: UM.Theme.getSize("base_unit").height * 12
+        Image
+        {
+            id: thumbnail
+            width: UM.Theme.getSize("toolbox_thumbnail_medium").width
+            height: UM.Theme.getSize("toolbox_thumbnail_medium").height
+            fillMode: Image.PreserveAspectFit
+            source: details.icon_url || "../images/logobot.svg"
+            anchors
+            {
+                top: parent.top
+                left: parent.left
+                leftMargin: UM.Theme.getSize("double_margin").width
+                topMargin: UM.Theme.getSize("double_margin").height
+            }
+        }
+
+        Label
+        {
+            id: title
+            anchors
+            {
+                top: thumbnail.top
+                left: thumbnail.right
+                leftMargin: UM.Theme.getSize("default_margin").width
+                right: parent.right
+                rightMargin: UM.Theme.getSize("double_margin").width
+                bottomMargin: UM.Theme.getSize("default_margin").height
+            }
+            text: details.name
+            font: UM.Theme.getFont("large")
+            wrapMode: Text.WordWrap
+            width: parent.width
+            height: UM.Theme.getSize("base_unit") * 2
```

Comment:
Not in my opinion. Then your type size starts to screw up your grid layout.

Reply:
But now your type size starts to screw up the readability since text will start to overlap if it's too large.

Reply:
Can't sacrifice the entire integrity of the grid for one line of text. If you want to make the text giant, you should also allot more space to that element.

Sorry but this is a change I'm not making. I don't think I was hired because I'm such a self-taught programmer but because I have formal training and professional experience as a designer. I'm not about to shoot Typography 101 in the face just for 1 less line of code. 😛